asset/api/Asset.Samples.Tests/AssetFixture.cs

```diff
@@ -28,7 +29,8 @@ public class AssetFixture : IDisposable, ICollectionFixture<AssetFixture>
     public string BucketName { get; }
     public RetryRobot Retry { get; } = new RetryRobot 
     { 
-        RetryWhenExceptions = new Type[] { typeof(XunitException) },
+        ShouldRetry = ex => ex is XunitException 
```

Comment:
I'd use more pattern matching here, assuming the SDK version allows you to:

```csharp
ShouldRetry = ex => ex is XunitException || ex is RpcException { StatusCode: StatusCode.DeadlineExceeded };
```

Reply:
Will try to see if it works. I have to fix a couple other more flakes anyway.